docs/paper/admin/reference/configuration/world-configuration.mdx

```diff
@@ -0,0 +1,740 @@
+        faster than the mob base speed, and -0.4 would be 40% slower'
+    disable-chest-cat-detection:
+      default: 'false'
+      description: 'Allows you to open chests even if they have a cat sitting on top
+        of them'
+    disable-creeper-lingering-effect:
+      default: 'false'
+      description: 'Disables creepers randomly leaving behind a lingering area effect
+        cloud'
+    disable-player-crits:
+      default: 'false'
+      description: 'Instructs the server to disable critical hits in PvP, treating
+        them as normal hits instead'
+    allow-spider-world-border-climbing:
+      default: 'true'
+      description: 'Whether spiders should be able to climb on the world border, as if it were a regular wall.'
+    door-breaking-difficulty:
+      default: '[HARD, NORMAL, EASY, PEACEFUL]'
+      description: 'Takes a list of difficulties at which each entity will attempt
+        to break doors'
+    ender-dragons-death-always-places-dragon-egg:
+      default: 'false'
+      description: 'Controls whether ender dragons should always drop dragon eggs
+        on death'
+    experience-merge-max-value:
+      default: '-1'
+      description: 'Instructs the server to put a maximum value on experience orbs,
+        preventing them all from merging down into 1 single orb. A value of -1 instructs
+        the server to use no max value, allowing them to merge down into a single
+        orb. This is especially noticeable when defeating boss monsters'
+    mobs-can-always-pick-up-loot:
+      default: 'false'
+      description: 'Controls whether zombies or skeletons (each configurable individually)
+        will always be able to pick up loot. If set to false, the probability that
+        a zombie picks up items depends on the world''s difficulty (vanilla behavior)'
+    nerf-pigmen-from-nether-portals:
+      default: 'false'
+      description: 'Removes AI from pigmen spawned via nether portals '
+    parrots-are-unaffected-by-player-movement:
+      default: 'false'
+      description: 'Makes parrots "sticky" so they do not fall off a player''s shoulder
+        when they move. Use crouch to shake them off'
+    phantoms-do-not-spawn-on-creative-players:
+      default: 'true'
+      description: 'Disables spawning of phantoms on players in creative mode '
+    phantoms-only-attack-insomniacs:
+      default: 'true'
+      description: 'Prevents phantoms from attacking players who have slept '
+    piglins-guard-chests:
+      default: 'true'
+      description: 'If piglins should attempt to guard chests when angered'
+    pillager-patrols:
+      disable:
+        default: 'false'
+        description: 'Disables Pillager patrols and associated AI'
+      spawn-chance:
+        default: '0.2'
+        description: 'Modify the spawn changes for patrols'
+      spawn-delay:
+        default: '12000'
+        description: 'Delay in ticks between spawn chance'
+      start:
```

Comment:
This is a section:
![image](https://github.com/PaperMC/docs/assets/41980282/8d17950e-39a3-479e-981a-299397a20776)